packages/eslint-config/src/baseConfig.ts

```diff
@@ -0,0 +1,273 @@
+      '@stylistic/no-mixed-operators': 'off',
+      '@stylistic/jsx-tag-spacing': 'off',
+      '@stylistic/jsx-function-call-newline': 'off',
+      '@stylistic/jsx-wrap-multilines': 'off',
+      '@stylistic/jsx-closing-tag-location': 'off',
+      '@stylistic/jsx-first-prop-new-line': 'off',
+      '@stylistic/jsx-closing-bracket-location': 'off',
+      '@stylistic/jsx-one-expression-per-line': 'off',
+      '@stylistic/jsx-max-props-per-line': 'off',
+      '@stylistic/jsx-curly-brace-presence': 'off',
+      '@stylistic/jsx-quotes': 'off',
+    },
+  },
+
+  // overrides for basic recommended rules, and custom rules
+  {
+    rules: {
+      'no-console': 'error',
+      'no-restricted-properties': [
+        'warn',
+        {
+          object: 'process',
+          property: 'geteuid',
+          message: 'process.geteuid() will throw on Windows. Do not use it unless you catch any potential errors.',
+        },
+        {
+          object: 'os',
+          property: 'userInfo',
+          message: 'os.userInfo() will throw when there is not an `/etc/passwd` entry for the current user (like when running with --user 12345 in Docker). Do not use it unless you catch any potential errors.',
+        },
+      ],
+      'no-restricted-syntax': [
+        // esquery tool: https://estools.github.io/esquery/
+        'warn',
+        {
+          // match sync FS methods except for `existsSync`
+          // examples: fse.readFileSync, fs.readFileSync, this.ctx.fs.readFileSync...
+          selector: `MemberExpression[object.name='fs'][property.name=/^[A-z]+Sync$/]:not(MemberExpression[property.name='existsSync']), MemberExpression[property.name=/^[A-z]+Sync$/]:not(MemberExpression[property.name='existsSync']):has(MemberExpression[property.name='fs'])`,
+          message: 'Synchronous fs calls should not be used in Cypress. Use an async API instead.',
+        },
+      ],
+      'padding-line-between-statements': [
+        'error',
+        {
+          'blankLine': 'always',
+          'prev': '*',
+          'next': 'return',
+        },
+        {
+          'blankLine': 'always',
+          'prev': [
+            'const',
+            'let',
+            'var',
+            'if',
+            'while',
+            'export',
+            'cjs-export',
+            'import',
+            'cjs-import',
+            'multiline-expression',
+          ],
+          'next': '*',
+        },
+        {
+          'blankLine': 'any',
+          'prev': [
+            'const',
+            'let',
+            'var',
+            'import',
+            'cjs-import',
+          ],
+          'next': [
+            'const',
+            'let',
+            'var',
+            'import',
+            'cjs-import',
+          ],
+        },
+      ],
+    },
+  },
+
+  {
+    // rules that are gold standard, but have many violations
+    // these are off while developing eslint, but should eventually be enabled
```

Comment:
in other words we need to disable these rules temporarily because there is an overwhelming amount of errors?